modules/telco-ran-lvms-operator.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-lvms-operator_{context}"]
+= LVMS Operator
+
+New in this release::
+* LVMS technical and user experience enhancements
+
+* Simplified LVMS `deviceSelector` logic
+
+* LVM Storage support for HyperShift hosted control plane and etcd use cases
+
+* LVM Storage with `ext4` and `PV` resources
+
+[NOTE]
+====
+LVMS Operator is an optional component.
+====
+
+Description::
+The LVMS Operator provides dynamic provisioning of block and file storage.
+The LVMS Operator creates logical volumes from local devices that can be used as `PVC` resources by applications.
+Volume expansion and snapshots are also possible.
++
+The following example configuration creates a `vg1` volume group that leverages all available disks on the node except the installation disk:
++
+.StorageLVMCluster.yaml
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: odf-lvmcluster
+  namespace: openshift-storage
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec: {}
+  storage:
+    deviceClasses:
+    - name: vg1
+      thinPoolConfig:
+        name: thin-pool-1
+        sizePercent: 90
+        overprovisionRatio: 10
+----
+
+Limits and requirements::
+* Ceph is not supported when used on cluster topologies with less than 3 nodes, for example, {sno} clusters or {sno} clusters with one worker node.
```

Comment:
Considering this is a section about LVMS, a comment on ceph feels rather out-of-place

Reply:
@imiller0 Can you comment on this please.

Reply:
I believe this is a leftover from when this operator was odf-lvm. It is OK to remove this line.